scripts/get-extism.sh

```diff
@@ -166,6 +166,8 @@ else
   reply=y
 fi
 if [ "$reply" = "y" ] || [ "$reply" = "Y" ] || [ "$reply" = "yes" ]; then
+  print "Creating $out_prefix if it doesn't exist"
```

Comment:
can we gate this print with a check if the path doesn't exist, and then print saying that we're making this directory?

Reply:
Done in 437eb7e and 1e12b2f